frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -182,7 +175,7 @@ const FileHandler = ({
         if (eventData) {
             trackAppInsightEvent(EventName.FILE_VALIDATOR, {
                 fileValidator: {
-                    schema: sender?.schemaName,
+                    schema: selectedSchemaOption?.value,
```

Comment:
This is being passed through as the BASE_SCHEMA_OPTIONS.value i.e.'csv-otc-covid-19'. Is that what we want or do we want to pass the key i.e. 'CSV_OTC'?
If we leave it as is then we will need to transform it within Azure.

Reply:
Yes, this is intentional because that's how the schemas are stored in memory -- see [Metadata.kt:202](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/src/main/kotlin/Metadata.kt#L202) where we're looking for the schemas.  `schemaStore` is a hash map whose keys are strings based on the schema filepaths like "hl7/hcintegrations-covid-19", "upload-covid-19", et cetera.

Regarding Azure, do you mean the App Insights side?  I thiiiiiink it should be fine to leave as-is since we're passing both the schema name and the format -- this'll give us granularity to see which schemas and file types are the most used/problematic.  What do you think?

Reply:
I'll show @chris-kuryak how the schema is returned in App Insights and let you know if we need to rework.